### ISSUE TITLE
fails to serialize Anahkiasen/polyglot

### ISSUE THREAD

**Issue by Remo** [jeremeamia/super_closure#51]
I was having a problem with confide which seems to be caused by a combination of super_close and polyglot. Original report can be found here: https://github.com/Zizaco/confide/issues/522#issuecomment-95816730

I can easily reproduce this if I install https://github.com/Anahkiasen/polyglot and then run this code:

```php
$lang = App::make('translator');
$closure = new \Illuminate\Support\SerializableClosure(function () use ($lang) { });
serialize($closure);
```

Which will show me `Serialization of 'Closure' is not allowed`

I like all the libraries I'm using, any idea on how to get them working together?

**Comment by Remo** [comment]
Okay it seems like it was fixed in version 2.. Any chance we can fix that for version 1 so that laravel 4.2 projects can use confide and polyglot too?

**Comment by GrahamCampbell** [comment]
Laravel 4.2 is not supported I'm afraid. If it was, I'd make sure laravel 4.2 allowed upgrading to the new super closure. I'd recommend upgrading to Laravel 5.1 since it's LTS and uses the newest version of super closure.
